recipes/batchspawner/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: batchspawner
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/batchspawner/batchspawner-{{ version }}.tar.gz
+  sha256: 9bae72f7c1bd9bb11aa58ecc3bc9fae5475a10fdd92dc0c0d67fa7eb95c9dd3a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  host:
+    # only targets unix systems, imports `pwd`
+    - __unix
+    - pip
+    - python >=3.6
+  run:
+    - __unix
+    - async_generator >=1.8
+    - jinja2
+    - jupyterhub-base >=0.5
+    - python >=3.6
+
+test:
+  imports:
+    - batchspawner
+  requires:
+    - pip
+    - packaging  # TODO: remove after https://github.com/conda-forge/jupyterhub-feedstock/issues/59
```

Comment:
Shouldn't this be under run? Seems like it's needed for the package.

Reply:
This one doesn't import `packaging`, and indeed, is only required by jupyterhub 2+

https://github.com/jupyterhub/jupyterhub/pull/3562

I've got a PR for that feedstock to fix:

https://github.com/conda-forge/jupyterhub-feedstock/pull/60